tests/integration/replication.tcl

```diff
@@ -1651,3 +1644,37 @@ start_server {tags {"repl external:skip"}} {
         }
     }
 }
+
+start_server {tags {"repl external:skip"}} {
+    # Replica
+    set replica [srv 0 client]
+    
+    start_server {} {
+        # Master
+        set master [srv 0 client]
+        set master_host [srv 0 host]
+        set master_port [srv 0 port]
+        
+        test "Test force full-sync" {
+            assert_equal 0 [s 0 sync_full]
+
+            # Expect error when master has no slaves
+            catch {$master debug force full-sync}
+
+            $replica replicaof $master_host $master_port
+            wait_for_sync $replica
+
+            # Test master
+            set prev [s 0 sync_full]
+            $master debug force-full-sync
+            wait_for_sync $replica
+            assert_equal [s 0 sync_full] [expr $prev+1]
```

Comment:
replica might not realize that it was disconnected by master? maybe we should use `wait_for_sync`([s 0 sync_full] == [expr $prev+1]) instead of `wait_for_sync`.

https://github.com/sundb/redis/actions/runs/14849419115/job/41690128502